Source/WebCore/Modules/webauthn/AuthenticationExtensionsClientInputs.idl

```diff
@@ -23,11 +23,20 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+ [
+    Conditional=WEB_AUTHN,
+] dictionary LargeBlobInputs {
```

Comment:
It seems to be called `AuthenticationExtensionsLargeBlobInputs` in the spec. Why are we using a different name?

Reply:
Since this is already nested under `AuthenticationExtensionsClientInputs` the longer name seemed redundant. I'm happy to use though if that's the preferred style.

Reply:
The IDL should match the spec ideally though you can use a shorter / different name on native side using [ImplementedAs] in IDL.

Reply:
Oh perfect, thanks!